packages/turbo-repository/scripts/test.mjs

```diff
@@ -0,0 +1,5 @@
+import { Workspace } from "../js/dist/index.js";
```

Comment:
Could this be in the tests here?
`packages/turbo-repository/__tests__/find.test.ts`

Or was this for debug?